experimenter/experimenter/theme/static_src/tailwind.config.js

```diff
@@ -1,58 +1,19 @@
-/**
- * This is a minimal config.
- *
- * If you need the full config, get it from here:
- * https://unpkg.com/browse/tailwindcss@latest/stubs/defaultConfig.stub.js
- */
-
 module.exports = {
-    content: [
-        /**
-         * HTML. Paths to Django template files that will contain Tailwind CSS classes.
-         */
-
-        /*  Templates within theme app (<tailwind_app_name>/templates), e.g. base.html. */
-        '../templates/**/*.html',
-
-        /*
-         * Main templates directory of the project (BASE_DIR/templates).
-         * Adjust the following line to match your project structure.
-         */
-        '../../templates/**/*.html',
-
-        /*
-         * Templates in other django apps (BASE_DIR/<any_app_name>/templates).
-         * Adjust the following line to match your project structure.
-         */
-        '../../**/templates/**/*.html',
-
-        /**
-         * JS: If you use Tailwind CSS in JavaScript, uncomment the following lines and make sure
-         * patterns match your project structure.
-         */
-        /* JS 1: Ignore any JavaScript in node_modules folder. */
-        // '!../../**/node_modules',
-        /* JS 2: Process all JavaScript files in the project. */
-        // '../../**/*.js',
-
-        /**
-         * Python: If you use Tailwind CSS classes in Python, uncomment the following line
-         * and make sure the pattern below matches your project structure.
-         */
-        // '../../**/*.py'
-    ],
-    theme: {
-        extend: {},
+  content: ["../../templates/**/*.html"],
+  theme: {
+    extend: {
+      colors: {
+        spaceCadet: "#242D4Eff",
+        lenurple: "#b590d4",
+        darkSlateBlue: "#4C4893ff",
+        rythm: "#7D68A7ff",
+      },
```

Comment:
will suggest having these colors constants somewhere globally stored in a constant file or something

Reply:
Isn't that what this is?

Reply:
oh I mean to have `colors.json` 
```json
{
  "spaceCadet": "#242D4Eff",
  "lenurple": "#b590d4",
  "darkSlateBlue": "#4C4893ff",
  "rythm": "#7D68A7ff"
}
```
and then 
```
const theme = {
  extend: {
    colors: colorDefinitions,
    // ... other theme settings
  }
};
```

Reply:
I was thinking of making a separate PR  for consolidating the tailwind config ,will it be okay if I include this in that one ?

Reply:
yup thats fine, just file a github issue for that

Reply:
Why do we need the colors in a separate JSON file?